requirements/edx/base.txt

```diff
@@ -132,7 +132,7 @@ future==0.18.2            # via edx-celeryutils, edx-enterprise, pyjwkest
 futures==3.3.0 ; python_version == "2.7"  # via django-pipeline, python-swiftclient, s3transfer, xblock-utils
 geoip2==2.9.0
 glob2==0.7
-gunicorn==19.9.0
+gunicorn==19.9.0 ; python_version == "2.7"
```

Comment:
This omits the `gunicorn` requirement altogether under Python 3, which we don't want.  I think the gunicorn version limit in `constraints.txt` should drop the environment marker again; we plan to run edx-platform in production under both Python 2 and 3 on different servers concurrently for a while, and it's probably best if we have the same `gunicorn` version on all the machines when we do that.  We can drop the constraint and upgrade again after we've fully switched to Python 3.

Reply:
@jmbowman Plz review again.